### PR TITLE
Update script to re-generate swagger.

### PR DESCRIPTION
**Pull request by brendandburns**
@nikhiljindal @zmerlynn 

This was broken when I tried to cut the 1.1.0 release.

### CODE REVIEW

File: build/mark-new-version.sh [hunk 1]
```diff
@@ -92,6 +92,10 @@ fi
 
 echo "+++ Running ./versionize-docs"
 ${KUBE_ROOT}/build/versionize-docs.sh ${NEW_VERSION}
+
+echo "+++ Updating swagger"
+${KUBE_ROOT}/hack/update-generated-swagger-docs.sh
```
Comment (nikhiljindal) [inline build/mark-new-version.sh]: I think it will be better to do this at the end of versionize-docs.
That is the script that is changing types.go and hence should regenerate swagger.

Replace hack/update-swagger-spec.sh by /hack/update-generated-swagger-docs.sh there.

**Comment by nikhiljindal** [comment]
cc @caesarxuchao

File: build/mark-new-version.sh (hunk 1 above)
Comment (caesarxuchao) [inline build/mark-new-version.sh]: Isn't this script already run once in versionize-docs.sh? https://github.com/kubernetes/kubernetes/blob/master/build/versionize-docs.sh#L74

Comment (brendandburns) [inline build/mark-new-version.sh]: @nikhiljindal done

@caesarxuchao that's `update-generated-docs.sh` not `update-generated-swagger-docs.sh`

**Comment by brendandburns** [comment]
Comments addressed, please re-check.

Thanks!
--brendan

File: build/mark-new-version.sh (hunk 1 above)
Comment (caesarxuchao) [inline build/mark-new-version.sh]: Sorry I missed that. 

I think we need to run `update-swagger-spec.sh` after `update-generated-swagger-docs.sh`.

File: build/versionize-docs.sh [hunk 2]
```diff
@@ -72,4 +72,5 @@ done
 $SED -ri -e "s|(releases.k8s.io)/[^/]+|\1/${NEW_VERSION}|" pkg/api/v[0-9]*/types.go
 
 ${KUBE_ROOT}/hack/update-generated-docs.sh
-${KUBE_ROOT}/hack/update-swagger-spec.sh
+${KUBE_ROOT}/hack/update-generated-swagger-docs.sh
+
```
Comment (caesarxuchao) [inline build/versionize-docs.sh]: I think we need to run hack/update-swagger-spec.sh after hack/update-generated-swagger-docs.sh

Comment (caesarxuchao) [inline build/versionize-docs.sh]: Could you also add hack/gen-swagger-doc/run-gen-swagger-docs.sh at last? That script will generate the docs/api-reference/definitions.html and operations.html. Thanks.

File: build/mark-new-version.sh (hunk 1 above)
Comment (nikhiljindal) [inline build/mark-new-version.sh]: update-generated-swagger-docs runs update-swagger-spec already?

File: build/versionize-docs.sh (hunk 2 above)
Comment (nikhiljindal) [inline build/versionize-docs.sh]: update-generated-swagger-docs.sh should call hack/gen-swagger-doc/run-gen-swagger-docs.sh.
It calls update-swagger-spec already.

Comment (brendandburns) [inline build/versionize-docs.sh]: @caesarxuchao I think that @nikhiljindal 's comment means this is unnecessary.  If that's true, can I have an LGTM?

Thanks!
--brendan

Comment (caesarxuchao) [inline build/versionize-docs.sh]: I think he means we should add hack/gen-swagger-doc/run-gen-swagger-docs.sh into update-generated-swagger-docs.sh.

But I guess we can do that in another PR.

Comment (nikhiljindal) [inline build/versionize-docs.sh]: Yes that is what I meant

**Comment by ihmccreery** [comment]
Closing in favor of #13339, which incorporates these changes, plus those requested by reviewers.
